webapp-django/crashstats/signature/static/signature/js/signature_tab_reports.js

```diff
@@ -97,11 +103,13 @@ SignatureReport.ReportsTab.prototype.getParamsForUrl = function () {
         });
     }
 
+    // Get the sort for the input.
+    params._sort = this.$sortInputHidden.val().trim().split(',') || [];
```

Comment:
Now I'm confused. 
if `this.$sortInputHidden.val().trim().split(',')` is exactly `['']` the end result is that `params._sort` will be `['']`. 
In node...
```
> [''] || []
[ '' ]
```
If you want `params._sort` to be `[]` when the `.val().trim()` is nothing, you need a different solution.